cwms-data-api/src/test/java/cwms/cda/api/location/kind/GateChangeScaleTest.java

```diff
@@ -77,6 +82,18 @@ class GateChangeScaleTest extends BaseOutletDaoIT {
     private static final TemporalAmount INTERVAL = Duration.of(5, ChronoUnit.MINUTES);
     private static final ZonedDateTime START = ZonedDateTime.of(2020, 1, 1, 0, 0, 0, 0, NumericalConstants.UTC_ZONEID);
     private static final ZonedDateTime END = START.plusYears(1).plus(INTERVAL);
+    private static final ZonedDateTime TS_START = ZonedDateTime.of(2024, 5, 31, 0, 0, 0, 0, NumericalConstants.UTC_ZONEID);
+    private static final Duration TS_DURATION = Duration.ofHours(120);
+
+    private static final String CWBI = "https://cwms-data-test.cwbi.us";
```

Comment:
Tests should not be pointed at a specific URL.
1. This is brittle, we cannot rely on cwbi-test services to be available; by definition even if in practice it usually is
2. It limits who can do the testing.

In the setup section default this to the test environment URL; if it's important to point it to another location at times allow a property override.